msteams-platform/bots/how-to/rate-limit.md

```diff
@@ -54,9 +54,10 @@ The following code shows an example of using exponential backoff using the trans
 public class BotSdkTransientExceptionDetectionStrategy : ITransientErrorDetectionStrategy
     {
         // List of error codes to retry on
-        List<int> transientErrorStatusCodes = new List<int>() { 429 };
+        private static readonly List<int> transientErrorStatusCodes = new List<int>() { 429 };
```

Comment:
No changes required for this line remove : private static readonly

Reply:
Updated.